docsets/structlog/docset.json

```diff
@@ -1,6 +1,6 @@
 {
     "name": "structlog",
-    "version": "22.1.0",
+    "version": "22.2.0",
     "archive": "structlog.tgz",
```

Comment:
This path points to `structlog.tgz`, but you added `structlog.tar.gz` instead. Please rename the archive to `structlog.tgz`